docs/src/main/asciidoc/_cloud/Getting_Started.adoc

```diff
@@ -0,0 +1,156 @@
+# WildFly on OpenShift - Developer Usage Guide
```

Comment:
Maybe we could rename the file "Getting_Started_on_OpenShift.adoc".

This doc focuses on building and installing WildFLy on OpenShift but eventually we should also address the same thing for vanilla Kubernetes instance.

Based on your work we could use the same doc structure, the difference would be that the user would configure the `wildfly-maven-plugin` to build the container image from Docker/podman and push it to a container repository. 
And then, they can use `helm` to deploy the application (without building it).